### PR TITLE
Using ip from order object

### PR DESCRIPTION
**Pull request by woutse**

### CODE REVIEW

**Review by IvoWams — APPROVED** [review]

**Review by daan-pay — APPROVED** [review]

**Review by max-geraci — APPROVED** [review]

File: app/addons/paynl_addon/paynl/classes/Pay/Api/Start.php [hunk 1]
```diff
@@ -242,7 +247,7 @@ protected function _getPostData() {
         }
 
         
-        $data['ipAddress'] = $_SERVER['REMOTE_ADDR'];
+        $data['ipAddress'] = $this->_ipaddress;
```
Comment (andypieters) [inline app/addons/paynl_addon/paynl/classes/Pay/Api/Start.php]: No check if the ipaddress is set?
Maybe add an if here, if its not set, retreive it from $_SERVER['REMOTE_ADDR'] like it was.

File: app/addons/paynl_addon/paynl/classes/Pay/Api/Start.php [hunk 2]
```diff
@@ -12,7 +12,8 @@ class Pay_Api_Start extends Pay_Api {
     private $_paymentOptionId;
     private $_paymentOptionSubId;
     private $_finishUrl;
-  
+    private $_ipaddress;
```
Comment (andypieters) [inline app/addons/paynl_addon/paynl/classes/Pay/Api/Start.php]: refactor to: $_ipAddress

**Review by andypieters — CHANGES_REQUESTED** [review]

File: app/addons/paynl_addon/paynl/classes/Pay/Api/Start.php (hunk 1 above)
Comment (woutse) [reply app/addons/paynl_addon/paynl/classes/Pay/Api/Start.php]: done

File: app/addons/paynl_addon/paynl/classes/Pay/Api/Start.php (hunk 2 above)
Comment (woutse) [reply app/addons/paynl_addon/paynl/classes/Pay/Api/Start.php]: done

**Review by andypieters — APPROVED** [review]

**Pull request merged**